include/bolgenos-ng/mem_utils.h

```diff
@@ -93,12 +98,7 @@ static inline void write_16(char *dest, char *src) {
 *	byte will be used.
 * \param size Size of memory to be set in bytes.
 */
-static inline void memset(void *mem, char val, size_t size) {
-	for(size_t pos = 0; pos != size; ++pos) {
-		write_8(((char *)mem) + pos, &val);
-	}
-}
-
+void memset(void *mem, char val, size_t size);
```

Comment:
Add note that function is needed for g++. Also it would be fine to know what else functions are expected to have by compiler.